site/content/docs/03-run-time.md

```diff
@@ -475,13 +475,48 @@ TODO
 * fade, fly, slide, scale, draw
 * crossfade...
 
-### `svelte/animation`
+### `svelte/animate`
 
-TODO
+The `svelte/animate` module exports one function for use with svelte [animations](docs#Animations).
+
+#### `flip`
+
+```sv
+animate:flip={params}
+```
+
+The `flip` function animates elements from their start position to their end position across `x` and `y` axes.
+
+`flip` accepts the following parameters:
+
+* `delay` (`number`, default 0) — milliseconds before starting
+* `duration` (`number` | `function`, default `d => Math.sqrt(d) * 120`) — see below
+* `easing` (`function`, default [`cubicOut`](docs#cubicOut)) — an [easing function](docs#svelte_easing)
+
+
+`duration` can be a `number` or a function `number => number`. When a number is passed, it is the duration in milliseconds that the animation lasts. When a function is passed, it must take the form `distance => duration`, receiving the distance the element will travel and returning the duration in milliseconds.
```

Comment:
this could be a bullet point, like below, and expand a bit on the function meaning.

Something like:

```
`duration` can be provided as:

- a `number` as the duration
- a `function`, `distance => duration`, receiving the distance the element will travel and returning the duration in milliseconds. Remember that the start and the end positions are calculated based on DOM and style, so if we know that an element can travel a lot in the screen we might want to use a function so that thr duration is related to the distance (for example the more we travel the higher the duration).
```